.github/workflows/check-for-dependency-updates.yaml

```diff
@@ -41,6 +41,8 @@ jobs:
       - name: Install Helm
         if: steps.update-grafana-agent.outputs.changed == 'true'
         uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
These are here to ensure that we're getting the latest stable version of the `helm` CLI
https://github.com/Azure/setup-helm?tab=readme-ov-file#example

Reply:
Without this, we were always getting helm 3.9, the latest is 3.14